app/jobs/application_job.rb

```diff
@@ -2,4 +2,18 @@ class ApplicationJob < ActiveJob::Base
   def humanized_name
     self.class.name
   end
+
+  def self.spawn_if_missing(world)
+    return if Foreman.in_rake? || Rails.env.test?
```

Comment:
There should be a condition, where we continue if `Foreman.in_rake?('dynflow:executor')`, as that's how one can run executor as well, especially in development.